src/app/test_executive/snark_test.ml

```diff
@@ -0,0 +1,150 @@
+open Core
```

Comment:
this entire test is just a stub/placeholder for now.  it will be completed the next PR